app/controllers/application_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  protect_from_forgery with: :exception
+  before_action :order_cart
+  before_action :find_user
```

Comment:
Jessica requested feedback: These are great controller filters to add to the applications controller.